Formula/lib/libmps.rb

```diff
@@ -0,0 +1,57 @@
+class Libmps < Formula
+  desc "Memory Pool System"
+  homepage "https://www.ravenbrook.com/project/mps/"
+  url "https://github.com/Ravenbrook/mps/archive/refs/tags/release-1.118.0.tar.gz"
+  sha256 "58c1c8cd82ff8cd77cc7bee612b94cf60cf6a6edd8bd52121910b1a23344e9a9"
+  license "BSD-2-Clause"
+
+  # Common build dependencies
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  # macOS specific dependencies
+  on_macos do
+    depends_on xcode: :build
+  end
```

Comment:
```suggestion
  depends_on xcode: :build
```
This doesn't need to be wrapped in an `on_macos` block.